esp/bindings/http/platform/httpservice.cpp

```diff
@@ -104,6 +104,7 @@ bool CEspHttpServer::rootAuth(IEspContext* ctx)
     EspHttpBinding* thebinding=getBinding();
     if (thebinding)
     {
+        Owned<IInterface> theBindingHolder = dynamic_cast<IInterface*>(thebinding);
```

Comment:
Dynamic casts without checking reslut for null will give coverity warnings. Is there any reason this cast needs to be a dynamic one?

Reply:
Dynamic cast is necessary because it's casting sideways. I'll add the check for nullptr.

Reply:
I don't think you have added the check yet.